### PR TITLE
Problem: insufficient validation of signed integers in CLI arguments

### LINKED ISSUE #332: Problem: insufficient validation of signed integers in CLI arguments
primarily in genaccounts.go and testnet.go.
e.g. in vesting, the values are parsed are signed integers and checked for non-zero, but the arguments are never checked for a negative values prior to use. Thus, inappropriate values may be sent into downstream logic.

### PR DESCRIPTION
**Pull request by devashishdxt**
Solution: Changed types from signed to unsigned integers. Fixes #332.

### CODE REVIEW

File: x/chainmain/client/cli/genaccounts.go [hunk 1]
```diff
@@ -82,24 +82,30 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 			if errstart != nil {
 				return fmt.Errorf("failed to parse vesting start: %w", errstart)
 			}
-			vestingStart, errstart := strconv.ParseInt(vestingStartStr, 10, 64)
+			vestingStartUint, errstart := strconv.ParseUint(vestingStartStr, 10, 64)
```
Comment (tomtau) [inline x/chainmain/client/cli/genaccounts.go:85]: should the bitsize be 63 instead of 64 (as it's later casted to a signed int64)?

**Review by tomtau — COMMENTED** [review]
not if this fix introduces potential overflows (when casting unsigned to signed numbers)

File: x/chainmain/client/cli/genaccounts.go (hunk 1 above)
Comment (devashishdxt) [reply x/chainmain/client/cli/genaccounts.go]: Changed to 32 to avoid overflow. Because we are converting it into `int64` later.

**Review by tomtau — COMMENTED** [review]
the integration test failed -- https://github.com/crypto-com/chain-main/pull/346/checks?check_run_id=1660486167#step:5:380 --- I'll try to rerun it with the latest master merged

**Review by tomtau — APPROVED** [review]

**Pull request merged**